src/main/java/de/tum/in/www1/artemis/repository/StatisticsRepository.java

```diff
@@ -36,6 +50,16 @@
             """)
     List<Map<String, Object>> getActiveUsers(@Param("startDate") ZonedDateTime startDate, @Param("endDate") ZonedDateTime endDate);
 
+    @Query("""
+            select s.submissionDate as day, u.login as username
+            from User u, Submission s, StudentParticipation p
+            where s.participation.id = p.id and p.student.id = u.id and s.submissionDate >= :#{#startDate} and s.submissionDate <= :#{#endDate} and u.login not like '%test%'
+            and p.exercise.id in :exerciseIds
+            order by s.submissionDate asc
+            """)
+    List<Map<String, Object>> getActiveUsersForCourse(@Param("startDate") ZonedDateTime startDate, @Param("endDate") ZonedDateTime endDate,
```

Comment:
please use a custom object here and not a map

Reply:
Done